server/fleet/software_installer.go

```diff
@@ -124,6 +124,9 @@ type SoftwareInstaller struct {
 	URL string `json:"url" db:"url"`
 	// FleetLibraryAppID is the related Fleet-maintained app for this installer (if not nil).
 	FleetLibraryAppID *uint `json:"-" db:"fleet_library_app_id"`
+	// AutomaticInstallPolicies is the list of policies that trigger automatic
+	// installation of this software.
+	AutomaticInstallPolicies []AutomaticInstallPolicy `json:"automatic_install_policies,omitempty" db:"-"`
```

Comment:
Double-checking the `omitempty`, not sure if we clearly decided one way or the other regarding empty arrays?

Reply:
@mna I did a quick search in our backend sync doc, looks like we did discuss returning an empty array if there are no results. I'm going to remove the `omitempty` based on that.